x-pack/legacy/plugins/alerting/server/routes/mute_all.ts

```diff
@@ -12,20 +12,18 @@ interface MuteAllRequest extends Hapi.Request {
   };
```

Comment:
muting! \o/ And even a `mute_all`!  Nice.

I'm sure we're going to need some query endpoints here, for the UI.  If there's not already a card or it's implemented elsewhere, seems card-worthy.

Reply:
Good point. I wonder now that KQL is supported on saved objects (API too I think?) if that will be sufficient for us. I've added this card to investigate: https://github.com/elastic/kibana/projects/26#card-27454227.